connectors/google/google-gemini/src/main/java/io/camunda/connector/gemini/model/GeminiRequestData.java

```diff
@@ -0,0 +1,350 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. Licensed under a proprietary license.
+ * See the License.txt file for more information. You may not use this file
+ * except in compliance with the proprietary license.
+ */
+package io.camunda.connector.gemini.model;
+
+import io.camunda.connector.feel.annotation.FEEL;
+import io.camunda.connector.generator.dsl.Property;
+import io.camunda.connector.generator.java.annotation.TemplateProperty;
+import jakarta.validation.constraints.NotNull;
+import java.util.List;
+
+public record GeminiRequestData(
```

Comment:
has the element template been reviewed by Yana? Just want to be sure as there's a lot of properties there :D

Reply:
This template fully displays the Gemini interface from Google Cloud, Christian looked at it earlier and he had no comments